internal/docker/client.go

```diff
@@ -332,8 +333,8 @@ func (c *Cli) startBuildkitSession(ctx context.Context, key string, syncedDirs [
 		dialSession := func(ctx context.Context, proto string, meta map[string][]string) (net.Conn, error) {
 			return c.Client.DialHijack(ctx, "/session", proto, meta)
 		}
-		_ = session.Run(ctx, dialSession)
-	}()
+		return session.Run(ctx, dialSession)
```

Comment:
All that errgroup plumbing for this one error capture!

Reply:
yep :grimacing: